src/lib/functions/registry.ts

```diff
@@ -322,6 +320,7 @@ export class FunctionsRegistry {
     if (event === 'buildError') {
       log(
         `${NETLIFYDEVERR} ${chalk.red('Failed to load')} function ${chalk.yellow(func?.displayName)}: ${
+          // @ts-expect-error buildError is unknown
```

Comment:
NIT: wouldn't it make more sense to do a type guard here?

```typescript
if (func?.buildError && typeof func.buildError === 'object' && 'message' in func.buildError) {
 // ...
}

Reply:
Happy to do that. Wanted to keep the diff small and easy to reason about, but agree a type guard is better here.

Reply:
done in https://github.com/netlify/cli/pull/6257/commits/4b7cbb6585444ff788629004663bb59b13297a6e